scene/gui/control.h

```diff
@@ -166,46 +166,46 @@ class Control : public CanvasItem {
 		Point2 pos_cache;
 		Size2 size_cache;
 		Size2 minimum_size_cache;
-		bool minimum_size_valid;
+		bool minimum_size_valid = false;
 
 		Size2 last_minimum_size;
-		bool updating_last_minimum_size;
+		bool updating_last_minimum_size = false;
 
-		float margin[4];
-		float anchor[4];
-		FocusMode focus_mode;
-		GrowDirection h_grow;
-		GrowDirection v_grow;
+		float margin[4] = {0};
```

Comment:
We could make the floatness explicit while at it (same for `rotation` and `expand`):
```suggestion
		float margin[4] = {0.0};
```